libs/libcommon/tests/test_backfill.py

```diff
@@ -362,6 +363,37 @@ def test_plan_compute_all(processing_graph: ProcessingGraph, up_to_date: list[st
     )
 
 
+@pytest.mark.parametrize(
+    "retries,max_retries,should_create",
+    [
+        (0, 1, True),
+        (3, 1, False),
+        (2, 4, True),
```

Comment:
```suggestion
        (3, 3, False),
        (2, 4, True),
```

Reply:
just to check we handle the limit correctly

Reply:
Done